projects/driver_controls/src/can_output.c

```diff
@@ -3,9 +3,25 @@
 #include "input_event.h"
 #include "log.h"
 
+#define CAN_OUTPUT_ID_POWER_STATE         5
```

Comment:
Where did these numbers come from?

Reply:
Those ids came from the confluence page. They can be changed as needed, but it's probably better to document where I got them in the code.

Reply:
Sounds good. @ckitagawa should be adding the output from our codegen somehow soon? (submodule or release branch? I think a new library would make sense)